client/src/components/SignIn/style.css

```diff
@@ -41,3 +41,419 @@ p {
   font-weight: 900;
   text-decoration: underline;
 }
+
```

Comment:
ooooo I had the LogoAnimation as it's own component because I was going to use a small version of it on the other pages, but we can just leave it like this right now and I can change it back to a component if I decide to use it that way.

Reply:
okay. yeah i suppose the only big deal here is where the css is for the animation. if we get to it sure, let's move it back.